scripts/core/data/users/index.tsx

```diff
@@ -0,0 +1,35 @@
+import {IUser} from 'superdesk-api';
+import {IAction} from '..';
+import UserActionTypes from './UserActionTypes';
+
+interface IUserState {
+    entities: {[id: string]: IUser};
+}
+
+const initialState: IUserState = {
+    entities: {},
+};
+
+export default function(state = initialState, action: IAction): IUserState {
```

Comment:
It's better to use a union type for `IAction`. That way, constants for actions are no longer needed and payloads are type-checked:

```
type IAction =
    {type: 'INIT_USERS', payload: string}
    | {type: 'UPDATE_USERS', payload: number};
```